include/boost/json/string.hpp

```diff
@@ -138,10 +138,6 @@ class string
         @par Complexity
 
         Constant.
-
-        @par Exception Safety
-
-        No-throw guarantee.
     */
     string() = default;
```

Comment:
But there's no `noexcept` here

Reply:
Yeah, I was like that when I got it. I'm going to investigate that to see if it will actually not throw.